WordPress/Classes/Services/ReaderTopicService.m

```diff
@@ -363,6 +363,10 @@ - (ReaderTopic *)siteTopicForPost:(ReaderPost *)post
     topic.topicDescription = post.blogDescription;
     topic.path = path;
 
+    [self.managedObjectContext performBlockAndWait:^{
```

Comment:
Huh. I shouldn't have used his handle. Thank you, mention notifications, i really appreciate that!